package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "frequency-style-guide",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "src/index.ts",
+  "types": "src/index.d.ts",
+  "scripts": {
+    "storybook": "storybook dev -p 6006 -s public",
+    "build-storybook": "storybook build",
+    "publish-package": "npm publish",
+    "dry-run": "npm pack --dry-run",
+    "lint": "npx eslint",
+    "lint:fix": "npx eslint --fix",
+    "format": "npm run prettier && npm run lint:fix",
+    "prettier": "npx prettier --write --plugin prettier-plugin-svelte --plugin prettier-plugin-tailwindcss src"
```

I don't think you need to run with `npx` when inside an npm script
```suggestion
    "lint": "eslint",
    "lint:fix": "eslint --fix",
    "format": "npm run prettier && npm run lint:fix",
    "prettier": "prettier --write --plugin prettier-plugin-svelte --plugin prettier-plugin-tailwindcss src"
```